controller/health.go

```diff
@@ -42,8 +43,10 @@ func setApplicationHealth(resources []managedResource, statuses []appv1.Resource
 			}
 		}
 		if healthStatus != nil {
-			resHealth := appv1.HealthStatus{Status: healthStatus.Status, Message: healthStatus.Message}
-			statuses[i].Health = &resHealth
+			if common.PersistResourceStatusHealth {
+				resHealth := appv1.HealthStatus{Status: healthStatus.Status, Message: healthStatus.Message}
+				statuses[i].Health = &resHealth
+			}
```

Comment:
I see that this opts the user into storing resource health in redis. Where is the change made to _stop_ the controller from storing the resource health in the CRD?